Adds `checkbox switch` attribute to `checkbox`  document. (#42831)
mdn/content: files/en-us/web/html/reference/elements/input/checkbox/index.md

- `value`
  - : The `value` attribute is one which all {{HTMLElement("input")}}s share; however, it serves a special purpose for inputs of type `checkbox`: when a form is submitted, only checkboxes which are currently checked are submitted to the server, and the reported value is the value of the `value` attribute. If the `value` is not otherwise specified, it is the string `on` by default. This is demonstrated in the section [Value](#value) above.

- `switch`

>   - : A [boolean](/en-US/docs/Glossary/Boolean/HTML) attribute that applies only to `checkbox` inputs. When present, it indicates that the `checkbox` represents an on/off `switch` rather than a traditional `checkbox`. It may affect the visual appearance of the `checkbox` control to resemble a `switch`, while the underlying behavior remains the same as normal `checkbox`.

Reviewer: ```suggestion
  - : A [boolean](/en-US/docs/Glossary/Boolean/HTML) attribute that applies only to `checkbox` inputs. When present, it indicates that the `checkbox` represents an on/off `switch` rather than a normal `checkbox`. It alters the appearance of the `checkbox` control, but the underlying behavior remains the same as that of a normal `checkbox`.
```

{{EmbedLiveSample('Checking_boxes_by_default', 600, 100)}}

### Switch as a checkbox

To make a checkbox look or act like an on/off switch in the supported browsers, you provide the `switch` attribute. See the below example:

```html
<form>
  <fieldset>
    <legend>Adjust your setting</legend>
    <div>
      <label for="theme">Dark mode</label>
      <input type="checkbox" name="theme" id="theme" switch checked />
    </div>
    <div>
      <label for="notifications">Notifications</label>
      <input type="checkbox" name="notifications" id="notifications" switch />
    </div>
    <button type="submit">Submit</button>
  </fieldset>
</form>
```

> [!NOTE]

> > Only some browsers render the checkbox as a switch, while the behavior is the same across all browsers.

Reviewer: ```suggestion
> While only some browsers render the checkbox as a switch, the behavior is the same across all browsers.
```
